### DISCUSSION TITLE
In node-server, how can I change port number?

### DISCUSSION THREAD

**Discussion by gitgitWi** [honojs/hono#1007] [Q&A] [answered]
Hello! 

Is there an official or genuine way to **change port number** in node server?

Actually, options argument of `serve` function has `port` property.

> _https://github.com/honojs/node-server/blob/e61a34aef5476c4efe28ed6e2467bdfe206cb6e3/src/server.ts#L18_

So I can change port number in my server like below, but it seems like kind of hack..

```typescript
serve(Object.assign(app, { port: 12345 }));
```

**Comment by yusukebe** [comment, marked as answer]
Hi @gitgitWi !

Sorry for the delay. You can write like follows:

```ts
serve({
  fetch: app.fetch,
  port: 12345,
})
```

Try it!
